src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/Maintainability/CodeMetricsComputationTests.cs

```diff
@@ -362,10 +362,10 @@ class C2 { }
 }";
 
             var expectedMetricsText = @"
-Assembly: (Lines: 7, ExecutableLines: 0, MntIndex: 100, CycCxty: 2, CoupledTypes: {N1.C1, System.Attribute}, DepthInherit: 2)
-   N1: (Lines: 7, ExecutableLines: 0, MntIndex: 100, CycCxty: 2, CoupledTypes: {N1.C1, System.Attribute}, DepthInherit: 2)
+Assembly: (Lines: 7, ExecutableLines: 2, MntIndex: 100, CycCxty: 2, CoupledTypes: {N1.C1, System.Attribute}, DepthInherit: 2)
```

Comment:
Interesting - so I presume this change is coming from consuming a new Microsoft.CodeAnalysis package referenced in tests, and is unrelated to this PR. We seem to be counting IOperation blocks for attributes as having 2 executable lines of code (basically 2 explicit operations in the operation block).

@333fred @Youssef1313 did we already change the operation tree for attributes in 4.1.0 version of MS.CA? The code for computing executable lines for an operation block for code metrics is here: https://github.com/dotnet/roslyn-analyzers/blob/086d1cc888ca0d8c96b94444935f488fcaf3fe98/src/Utilities/Compiler/CodeMetrics/ComputationalComplexityMetrics.cs#L87-L355

Reply:
It's not merged yet.